internal/cache/texture_common.go

```diff
@@ -45,6 +65,10 @@ func RangeExpiredTexturesFor(canvas fyne.Canvas, f func(fyne.CanvasObject)) {
 // gl context to ensure textures are deleted from gl.
 func RangeTexturesFor(canvas fyne.Canvas, f func(fyne.CanvasObject)) {
 	textures.Range(func(key, value any) bool {
+		if _, ok := key.(FontCacheEntry); ok {
+			return true // TODO what?
```

Comment:
Should this TODO be addressed before merge?

Reply:
oops!

Reply:
Updated - a doc change covered it I think.
If the API is needed for other purposes at some point then we may need to track all of the objects referencing a certain string but it's not needed yet and because this will evolve I'm leaving it minimal where possible.